src/main/java/com/powsybl/openloadflow/network/LfNetwork.java

```diff
@@ -365,6 +372,15 @@ public LfLoad getLoadById(String id) {
         return loadsById.get(id);
     }
 
+    public List<LfArea> getAreas() {
+        return new ArrayList<>(areasById.values());
```

Comment:
The copy here is expensive, not necessary,  and will require unnecessary GC work.
Can't you instead create a getAreasStream  and hasArea() to replace getAreas().isEmpty() ?

Reply:
Resolved